### PR TITLE
facs: analyze samples scroll bar and run samples btn

### LINKED ISSUE #171: NSF Exercise 3 - Change scroll bar in Samples window on Flow Cytometry Analyze page
The scroll bar should always be visible to indicate to users that there are more than 12 samples (like the samples window on the load/analyze page of the microscopy experimental technique). In addition, the scroll bar should appear further over to the right so that it doesn't overlap the name of the sample.
https://scbdev.odl.mit.edu/#view=facs&assignment_id=scb_ex3&experiment_id=scb_Experiment_56j9bo28ea87j&facs_id=scb_Facs_j7im5mibq9dn

### PR DESCRIPTION
**Pull request by annagav**
Fix #171

### CODE REVIEW

File: html_app/ui/facs.gss [hunk 1]
```diff
@@ -96,6 +95,15 @@
 	position: absolute;
 	bottom: 84px;
 }
+ol.scb_s_facs_choose_samples_order_list::-webkit-scrollbar {
+    -webkit-appearance: none;
+    width: 7px;
+}
```
Comment (pdpinch) [inline html_app/ui/facs.gss:102]: Why webkit only?

**Comment by pdpinch** [comment]
I've been testing everything in Chrome because my understanding was that we weren't supporting Firefox. 

In Chrome, the scrollbars for samples aren't working for me as soon as I click "Run Samples"

screen shot 2015-07-23 at 8 27 04 am

**Comment by annagav** [comment]
Fixed!

**Comment by pdpinch** [comment]
Looks good. What was wrong with the earlier version?

**Pull request merged**